src/main.c

```diff
@@ -402,7 +402,7 @@ void load_config(void)
 			loader_users_check();
 		loaded = true;
 	} else if (!loaded) {
-		die("cannot load config file");
+		die("cannot load config file %s", cf_config_file);
```

Comment:
This is okay. We should probably quote the file name, like in other messages. Also, there is a `log_warning` below that should perhaps be adjusted in the same way?